plugins/ProfessionalServices/tests/UI/ProfessionalServices_PluginPromo_spec.js

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * Screenshot integration tests.
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe("ProfessionalServices_PluginPromo", function () {
+    this.timeout(0);
+
+    var generalParams = 'idSite=1&period=day&date=2017-01-02',
+        urlBase = '?module=CoreHome&' + generalParams + '&action=index&';
+
+    before(function () {
+        testEnvironment.pluginsToLoad = ['PrivacyManager', 'Marketplace', 'ProfessionalServices'];
+        testEnvironment.save();
+    });
+
+    it('should load A/B Testing plugin promo detail view', async function() {
+        const category = 'ProfessionalServices_PromoAbTesting';
+        const subcategory = 'ProfessionalServices_PromoOverview';
+
+        await page.goto(urlBase + 'category=' + category + '&subcategory=' + subcategory);
+        await page.waitForNetworkIdle();
+        expect(await page.screenshotSelector('.pageWrap, .pluginPromo')).to.matchImage('promo_abtesting');
```

Comment:
Can we limit the screenshots to just `.pluginPromo`?

The `.pageWrap` seems to not add much value but much grey space, probably matching the height of the navigation.

Reply:
Yeah, I can remove that, it's a copy from some other UI test. The images are likely taller than they need to be because of that.